packages/cli/src/util/telemetry/README.md

```diff
@@ -0,0 +1,5 @@
+# Vercel CLI Telemetry
```

Comment:
Intentionally left blank. Since the review of this PR may change how this works, I'm defering this to a follow up PR